sql/mssql/00020-Evaluation-Tables.sql

```diff
@@ -0,0 +1,149 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE TABLE eppeta.PerformanceEvaluation (
+    EducationOrganizationId INT NOT NULL,
+    EvaluationPeriodDescriptorId INT NOT NULL,
+    PerformanceEvaluationTitle VARCHAR(50) NOT NULL,
```

Comment:
I agree with use of `VARCHAR`. Just an opportunity to comment: any database that needs to support internationalization would need `NVARCHAR` for support of UTF-16. Since Ed-Fi is only for the US market, that kind of internationalization for support of highly extended character sets seems very unlikely, so `VARCHAR` is safe for now.

Reply:
Not sure how these ended up as varchar, but they should be nvarchar, @StevenM0 please update any VARCHAR to NVARCHAR

Reply:
@stevenarnold-ed-fi why would we use NVARCHAR here? I don't mean to make a big deal about it, but I don't see a compelling reason for it.

Reply:
@stevenarnold-ed-fi do you want to update to `NVARCHAR` in this PR?

Reply:
Updated for consistency with authentication tables